includes/admin/builder/class-evf-builder-settings.php

```diff
@@ -247,128 +255,163 @@ public function output_content() {
 			if ( isset( $settings['email']['evf_send_confirmation_email'] ) && '1' === $settings['email']['evf_send_confirmation_email'] ) {
-				$settings['email'][ $unique_connection_id ] = array( 'connection_name'   => esc_html__( 'User Notification', 'everest-forms' ) );
+				$settings['email'][ $unique_connection_id ] = array( 'connection_name' => esc_html__( 'User Notification', 'everest-forms' ) );
 
 				foreach ( $email_settings as $email_setting ) {
 					$settings['email'][ $unique_connection_id ][ $email_setting ] = isset( $settings['email'][ $email_setting ] ) ? $settings['email'][ $email_setting ] : '';
 				}
 			}
 		}
 
-		$form_name = isset( $settings['form_title'] ) ? ' - '. $settings['form_title'] : '';
+		$form_name = isset( $settings['form_title'] ) ? ' - ' . $settings['form_title'] : '';
 
 		echo '<div class="evf-content-section evf-content-email-settings">';
 		echo '<div class="evf-content-section-title">';
 		_e( 'Email', 'everest-forms' );
 		echo '</div>';
 
 		foreach ( $settings['email'] as $connection_id => $connection ) :
-			if ( preg_match( '/connection_/' , $connection_id ) ) {
-				echo '<div class="evf-content-email-settings-inner" data-connection_id='.$connection_id.'>';
+			if ( preg_match( '/connection_/', $connection_id ) ) {
+				echo '<div class="evf-content-email-settings-inner" data-connection_id=' . $connection_id . '>';
 
 				everest_forms_panel_field(
 					'text',
-					'settings[email]['.$connection_id.']',
+					'settings[email][' . $connection_id . ']',
 					'connection_name',
 					$this->form_data,
 					'',
 					array(
-						'default' => isset( $settings['email'][$connection_id]['connection_name'] ) ? $settings['email'][$connection_id]['connection_name'] : __('Admin Notification', 'everest-forms'),
+						'default' => isset( $settings['email'][ $connection_id ]['connection_name'] ) ? $settings['email'][ $connection_id ]['connection_name'] : __( 'Admin Notification', 'everest-forms' ),
 						'class'   => 'everest-forms-email-name',
 					)
 				);
 				everest_forms_panel_field(
 					'text',
-					'settings[email]['.$connection_id.']',
+					'settings[email][' . $connection_id . ']',
 					'evf_to_email',
 					$this->form_data,
 					__( 'To Address', 'everest-forms' ),
 					array(
-						'default'    => isset( $settings['email'][ $connection_id ]['evf_to_email'] ) ? $settings['email'][ $connection_id ]['evf_to_email'] : '{admin_email}',
-						'tooltip'    => __( 'Enter your email address to receive notifications; separate with a comma if multiple addresses.', 'everest-forms' ),
-						'smarttags'  => array(
+						'default'   => isset( $settings['email'][ $connection_id ]['evf_to_email'] ) ? $settings['email'][ $connection_id ]['evf_to_email'] : '{admin_email}',
+						'tooltip'   => __( 'Enter your email address to receive notifications; separate with a comma if multiple addresses.', 'everest-forms' ),
+						'smarttags' => array(
+<<<<<<< HEAD
```

Comment:
@ujw0l I think you haven't tested the code after merging the develop branch. Here I can see the merge conflicts.

Reply:
@roshansameer fixed :)